src/composition.rs

```diff
@@ -566,29 +477,26 @@ impl Composition {
     ///
     /// This must be called before any other composition can be made, as it'll set up the
     /// statement that will be managed by composition.
+    ///
+    /// Note: this call is destructive! If there are statements in this file already, they'll be removed and
+    /// replaced with the composition statement.
```

Comment:
Does that mean if users write their own statements with any different bucket, measurement, fields, or tag_values, then their statements will be replaced?

Reply:
Yes, if there is an expression statement already in the file, it will be removed on init. This  means the UI has to know and keep track of that to turn composition off. @garylfowler was okay with this pattern.